chapters/planning-data-work.tex

```diff
@@ -151,162 +469,185 @@ \subsection{Code management}
-
-\subsection{Version control}
-
-A \textbf{version control system} is the way you manage the changes to any computer file.
-This is important, for example, for your team to be able to find the version of a presentation that you delivered to your donor,
-but also to understand why the significance level of your estimates has changed.
-Everyone who has ever encountered a file named something like \texttt{final\_report\_v5\_LJK\_KLE\_jun15.docx}
-can appreciate how useful such a system can be.
-Most file sharing solutions offer some level of version control.
-These are usually enough to manage changes to binary files (such as Word and PowerPoint documents) without needing to appeal to these dreaded file names.
-For code files, however, a more complex version control system is usually desirable.
-We recommend using Git\sidenote{\textbf{Git:} a multi-user version control system for collaborating on and tracking changes to code as it is written.} for all plain text files.
-Git tracks all the changes you make to your code,
-and allows you to go back to previous versions without losing the information on changes made.
-It also makes it possible to work on two parallel versions of the code,
-so you don't risk breaking the code for other team members as you try something new,
-
-Increasingly, we recommend the entire data work folder
-to be created and stored separately in GitHub.
-Nearly all code and outputs (except datasets) are better managed this way.
-Code is written in its native language,
-and it's becoming more and more common for written outputs such as reports,
-presentations and documentations to be written using different \textbf{literate programming}
-tools such as {\LaTeX} and dynamic documents.
-You should therefore feel comfortable having both a project folder and a code folder.
-Their structures can be managed in parallel by using \texttt{iefolder} twice.
-The project folder can be maintained in a synced location like Dropbox,
-and the code folder can be maintained in a version-controlled location like GitHub.
-While both are used for sharing and collaborating,
-there is a sharp difference between the functionality of sync and version control.
-Namely, sync forces everyone to have the same version of every file at all times
-and does not support simultaneous editing well; version control does the opposite.
-Keeping code in a version-controlled folder will allow you
-to maintain better control of its history and functionality,
-and because of the specificity with which code depends on file structure,
-you will be able to enforce better practices there than in the project folder.
+One other important advantage of code review if that
+making sure that the code is running properly on other machines,
+and that other people can read and understand the code easily,
+is the easiest way to be prepared in advance for a smooth project handover.
```

Comment:
This is more than project handover, right? What we have said here also applies to projects where a single PhD candidate works on a paper that no one else ever will work on? Doesn't it have more to do with Research Transparency?

I guess it is Research Transparency in all projects and also project handover in collaborative projects.

Reply:
92f4a942d26172f943ee9fe9f1359f6ae3555547